tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py

```diff
@@ -94,275 +94,276 @@
-                    required=False, validators=[
+                    friendly_name="race american indian", required=False, validators=[
                         validators.isInLimits(0, 2)
                     ]),
               Field(item="30C", name='RACE_ASIAN', type='number', startIndex=40, endIndex=41,
-                    required=False, validators=[
+                    friendly_name="race asian", required=False, validators=[
                         validators.isInLimits(0, 2)
                     ]),
               Field(item="30D", name='RACE_BLACK', type='number', startIndex=41, endIndex=42,
-                    required=False, validators=[
+                    friendly_name="race black", required=False, validators=[
                         validators.isInLimits(0, 2)
                     ]),
               Field(item="30E", name='RACE_HAWAIIAN', type='number', startIndex=42, endIndex=43,
-                    required=False, validators=[
+                    friendly_name="race hawaiian", required=False, validators=[
                         validators.isInLimits(0, 2)
                     ]),
               Field(item="30F", name='RACE_WHITE', type='number', startIndex=43, endIndex=44,
-                    required=False, validators=[
+                    friendly_name="race white", required=False, validators=[
                         validators.isInLimits(0, 2)
                     ]),
               Field(item="31", name='GENDER', type='number', startIndex=44, endIndex=45,
-                    required=True, validators=[
+                    friendly_name="gender", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0)
                     ]),
               Field(item="32A", name='FED_OASDI_PROGRAM', type='number', startIndex=45, endIndex=46,
-                    required=True, validators=[
+                    friendly_name="federal qasdi program", required=True, validators=[
                         validators.oneOf([1, 2])
                     ]),
               Field(item="32B", name='FED_DISABILITY_STATUS', type='number', startIndex=46, endIndex=47,
-                    required=True, validators=[
+                    friendly_name="federal disability status", required=True, validators=[
                         validators.oneOf([1, 2])
                     ]),
               Field(item="32C", name='DISABLED_TITLE_XIVAPDT', type='number', startIndex=47, endIndex=48,
-                    required=True, validators=[
+                    friendly_name="disabled title xivapdt", required=True, validators=[
                         validators.oneOf([1, 2])
                     ]),
               Field(item="32D", name='AID_AGED_BLIND', type='number', startIndex=48, endIndex=49,
-                    required=False, validators=[
+                    friendly_name="receives aid to the aid aged blind", required=False, validators=[
```

Comment:
receives aid from the aid aged blind?

Reply:
it is correct: Receives Aid to the Aged, Blind, and Disabled Under Title XVI-AABD of the Social Security Act